.github/actions/npm-blacklist/action.yml

```
@@ -0,0 +1,42 @@
+  additional-blacklist-file:
+    description: 'Path to an additional blacklist file with list of pkg@version per line, example: ./my-blacklist.txt'
+    required: false
+    default: ''
+  additional-blacklist-pkgs:
+    description: 'Multiline list of additional blacklisted packages, example: pkg@version'
+    required: false
+    default: ''
+  override-blacklist:
+    description: 'Path to blacklist file to override the default known blacklist'
+    required: false
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)